bundle-size/api.js

```diff
@@ -181,6 +218,54 @@ function extraBundleSizesSummary(otherBundleSizeDeltas, missingBundleSizes) {
   );
 }
 
+/**
+ * Choose the set of all the potential approver teams into a single team.
+ *
+ * Could end up choosing the fallback set defined in the `.env` file, even if
+ * the fallback set is not in the input.
+ *
+ * @param {Array<Array<string>>} allPotentialApproverTeams all the potential
+ *   teams that can approve this pull request.
+ * @param {Logger} log logging function/object.
+ * @param {number} pullRequestId the pull request ID.
+ * @return {Array<string>} the selected subset of all potential approver teams
+ *   that can approve this bundle-size change.
+ */
+function choosePotentialApproverTeams(
+  allPotentialApproverTeams,
+  log,
+  pullRequestId
+) {
+  let potentialApproverTeams;
+  switch (allPotentialApproverTeams.length) {
+    case 0:
+      potentialApproverTeams = [];
+      log(
+        `Pull request #${pullRequestId} does not require ` +
+          'bundle-size approval'
+      );
+      break;
+
+    case 1:
+      potentialApproverTeams = allPotentialApproverTeams[0];
```

Comment:
Shouldn't this be an array?

Reply:
It is! `allPotentialApproverTeams` is an `!Array<!Array<string>>`